fiftyone/core/dataset.py

```diff
@@ -1898,7 +1901,7 @@ def create_summary_field(
 
         if create_index:
             for _field_name in index_fields:
-                self.create_index(_field_name)
+                self.create_index(_field_name, wait=wait)
```

Comment:
Have you benchmarked the cost of `create_index()` vs `_populate_summary_field()`? Note that we create the index *before* populating the field here.

I bet running the aggregation to populate the field takes longer, which would mean that `create_summary_field(wait=False)` would not actually return fast.

Reply:
That's my fear too; I hope that at least setting `wait=False` will allow the execution of this function to terminate a little bit earlier...

Reply:
Probably the right approach is allowing the users to schedule summary field operations, but it's currently out of scope for v2.2.0; one thing we can do from UI/UX side is to add a user warning to summary field operations so at least the users are aware of it.